CHANGELOG.md

```diff
@@ -3,17 +3,26 @@
 ## 24.2.0-SNAPSHOT
 
 ### Breaking Changes
+- SNAP - Snap sync is now the default for named networks [#6530](https://github.com/hyperledger/besu/pull/6530)
+  - if you want to use the previous default behavior, you'll need to specify `--sync-mode=FAST`
+- BONSAI - Default data storage format is now Bonsai [#6536](https://github.com/hyperledger/besu/pull/6536)
+  - if you had previously used the default (FOREST), at startup you will get an error indicating the mismatch
+    `Mismatch: DB at '/your-path' is BONSAI (Version 2) but config expects FOREST (Version 1). Please check your config.`
```

Comment:
Should be the other way around?

```suggestion
    `Mismatch: DB at '/your-path' is FOREST (Version 1) but config expects BONSAI (Version 2). Please check your config.`
```

Reply:
it seems misleading, because your existing db is Forest, anyway that will change in the database metadata refactoring [PR](https://github.com/hyperledger/besu/pull/6555)

Reply:
right. corrrect